src/js/components/post/Post.ts

```diff
@@ -133,6 +134,9 @@ export class Post implements PostData {
             "buttonsVote", "buttonsFav",                        // renderButtons
         ]);
 
+        // Get upvoteOnFavorite setting from PostViewer
+        conf.upvoteOnFavorite = ModuleController.get(PostViewer).fetchSettings("upvoteOnFavorite");
```

Comment:
Fetching settings from multiple different modules is a little messy, but it's not the end of the world.
If more settings will need to be fetched from PostViewer, this could be streamlined as

``` JS
const conf = {
    ...ModuleController.get(BetterSearch).fetchSettings([
        "imageRatioChange",                                 // renderArticle
        "clickAction",                                      // renderLink
        "imageLoadMethod", "hoverTags",                     // renderImage
        "autoPlayGIFs", "maxPlayingGIFs",
        "ribbonsFlag", "ribbonsRel",                        // renderRibbons
        "buttonsVote", "buttonsFav",                        // renderButtons
    ]),
    ...ModuleController.get(PostViewer).fetchSettings([
        "upvoteOnFavorite"
    ]),
};
```

Reply:
It does not make much of a difference right now.

Reply:
Alright, I'll just resolve this then.